### PR TITLE
fix(contactwindow):removed pin link per request

### PR DESCRIPTION
**Pull request by akonsowski**
@blynch11p @rkenyon1969 @lsemesky

### CODE REVIEW

File: app/js/components/contacts/ContactsWindow.jsx [hunk 1]
```diff
@@ -64,10 +64,10 @@ var ContactInfo = React.createClass({
                         </div>
                         <div className="col-md-9">
                             <h3>Connect With Us</h3>
-                            <p>Want to know the latest? Stay up to date with us through any of our social networks. We'll keep you in the loop.</p>
+                            <p>Want to know the latest? Stay up to date with us through any of our social networks. We will keep you in the loop.</p>
                             <span className="btn-row">
                                 <a className="btn btn-primary" href={SOCIAL_CHIRP_ADDRESS} target={this.getTarget(SOCIAL_CHIRP_ADDRESS)}>Chirp</a>
-                                <a className="btn btn-primary" href={SOCIAL_PIN_ADDRESS} target={this.getTarget(SOCIAL_PIN_ADDRESS)}>Pin</a>
+                                //<a className="btn btn-primary" href={SOCIAL_PIN_ADDRESS} target={this.getTarget(SOCIAL_PIN_ADDRESS)}>Pin</a>
```
Comment (blynch11p) [inline app/js/components/contacts/ContactsWindow.jsx]: This shouldn't work (and doesn't for me).  React Comments are {/* <code> */}

**Comment by blynch11p** [comment]
:-1:  See comment above.

File: app/js/components/contacts/ContactsWindow.jsx (hunk 1 above)
Comment (akonsowski) [inline app/js/components/contacts/ContactsWindow.jsx]: yea it looks like we will have to take this part out of the render all together

**Comment by akonsowski** [comment]
@blynch11p that bracket seems to have fixed the error that the render was throwing

**Comment by blynch11p** [comment]
Better
:+1:

**Pull request merged**